test/unit/utils/errors.spec.js

```diff
@@ -79,7 +79,24 @@ test('new invalidValueError error can be created with custom message, status and
     const message = 'ERROR'
     const status = 1000
     const cause = {}
-    const error = errors.newInvalidValueError(message, status, cause)
+    const error = errors.newInvalidValueError(message, cause, status)
+    expect(error.message).toBe(message)
+    expect(error.status).toBe(status)
+    expect(error.cause).toEqual(cause)
+})
+
+test('new entityAlreadyCreated error can be created with default message, status and cause', () => {
```

Comment:
usemos it o test pero no las 2 , unifiquemos :d

Reply:
Pasa que el it tiene ma sentido cuando está dentro del describe, y el test cuando está por fuera ya que vive solo. La misma oración te lleva a llamar it a la forma de testear o test.. igual podemos llegar a un criterio en comun